### DISCUSSION TITLE
Is there a way to add a color to a MultiChoice field type via Add-PnPField

### DISCUSSION THREAD

**Discussion by JordanMurrayFedSav** [pnp/powershell#857] [Ideas]
Just wondering if there is currently a way to assign a color to a choice for choice or multichoice field types using Add-PnPField. I didn't see anything in the documentation but with the capability in SharePoint I was hoping I'd be able to assign colors to it  by doing something like

`Add-PnPField -List Progress_List -DisplayName "Tasks" -InternalName "Progress" -Type MultiChoice -AddToDefaultView -Choices "Started | Yellow", "Need Help | Red", "Awaiting Review | Blue", "Done | Green".`

Maybe not exactly like that but to give an idea.

**Comment by BaronSparky** [comment]
Have you looked at https://github.com/pnp/PnP-PowerShell/issues/2241

I think you can also use the "customFormatter" property for fields in the same way.
